packages/next/site/src/pages/components/button.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ComponentDocs from '../../components/componentDocs';
+import PropTypes from 'prop-types';
+import { Button, ButtonVariant, Title } from '@patternfly/react-core';
+
+const propTypes = {
+  data: PropTypes.any.isRequired
+};
+
+const ButtonDocs = ({ data }) => (
+  <ComponentDocs data={data}>
+    <div>
+      <Title size="lg" withMargins>
+        Variants
+      </Title>
+      {Object.values(ButtonVariant).map(variant => (
+        <span
```

Comment:
any reason to wrap them w/ spans vs displaying them like they are displayed on the  [pf-next docs](https://pf-next.com/components/Button/examples/)?

Reply:
pf-next is adding spaces in between.   Could do this with React here I suppose, but I figured css with margins would be a better way to separate them. That said, I should have just passed the extra class to them instead of wrapping.